help/blocks/columns/columns.css

```diff
@@ -2,13 +2,18 @@
   background-color: var(--background-color);
   font-size: var(--body-font-size-xs);
   word-break: break-word;
+  margin-bottom: 64px;
```

Comment:
this should possibly go on `.columns-wrapper`
as for now the effect isn't visible, I tested on link: https://article-columns--macquarie-help-centre--hlxsites.hlx.page/help/advisers/wrap-tax-and-eofy/eofy-guides-and-resources/tax-resources-and-guides

Reply:
right! it was working before I added the rule for `:last-child` to prevent white space at the end. Adjusting now.